pkg/export/template.go

```diff
@@ -147,20 +149,36 @@ func templateJoin(sep string, input []interface{}) (string, error) {
 }
 
 func (t *Template) row(fields ...interface{}) (string, error) {
+	if t.tablePrinter == nil {
+		tablePrinter := utils.NewTablePrinter(t.io)
+		t.tablePrinter = &tablePrinter
+	}
 	for _, e := range fields {
 		s, err := jsonScalarToString(e)
 		if err != nil {
 			return "", fmt.Errorf("failed to write row: %v", err)
 		}
-		t.tablePrinter.AddField(s, nil, nil)
+		(*t.tablePrinter).AddField(s, nil, nil)
```

Comment:
How come dereferencing of `*t.tablePrinter` is needed here to call its methods?

Reply:
The editor was showing a syntax error otherwise, but I [see now](https://play.golang.org/p/KN3PYXJSGYk) it's not actually necessary.

Reply:
Seems it is required for a pointer to an interface, which `TablePrinter` is, of course:

![image](https://user-images.githubusercontent.com/1532486/128748862-ae3d2969-ea66-4005-b25a-212d960e4b9b.png)